internal/endpoints/endpointsFetcher.go

```diff
@@ -0,0 +1,60 @@
+package endpoints
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+// Schema for data pulled out of EndpointSources file
+type ListOfEndpoints struct {
+	Entries []struct {
+		OrganizationName     string `json:"OrganizationName"`
```

Comment:
does it make sense to just store a namespacified version of organization name in the structure?

Reply:
We _could_ store the namespacified string, however using the OrgName as the namespace was part of the first-pass solution and the OrgNames aren't even being used as their own namespaces anymore -- the namespace is now just "AllEndpoints" whereas in the past I had given _each_ Organization their own namespace. I can remove the namespaceification code if the "AllEndpoints" namespace is the direction we are going.